app/apps/client/admin/helpers.js

```diff
@@ -363,11 +363,12 @@ export const appStatusSpanProps = ({
 
 export const formatPrice = (price) => `\$${ Number.parseFloat(price).toFixed(2) }`;
 
-export const formatPricingPlan = ({ strategy, price, tiers }) => {
+export const formatPricingPlan = ({ strategy, price, tiers = [] }) => {
 	const { perUnit = false } = (Array.isArray(tiers) && tiers.find((tier) => tier.price === price)) || {};
 
 	const pricingPlanTranslationString = [
 		'Apps_Marketplace_pricingPlan',
+		tiers.length > 0 && 'startingAt',
```

Comment:
just to be safe, should it be tested here if `tiers` is an array just like on like 367?

Reply:
Probably but will it look ugly?

Reply:
```suggestion
		Array.isArray(tiers) && tiers.length > 0 && 'startingAt',
```

Reply:
the latter seems prettier =)

Reply:
committed then :)